tests/core/abi-utils/test_abi_utils.py

```diff
@@ -1507,7 +1507,7 @@ def test_get_aligned_abi_inputs(
             },
             TypeError,
             'Expected non-string sequence for "tuple[]" component type: got '
-            "{(9, 10), (5, 6), (7, 8)}",
+            f"{set(sorted({(9, 10), (5, 6), (7, 8)}))}",
```

Comment:
Hmm I feel like this might still get flakiness somehow... calling `set()` on `sorted()` and relying on that plus `repr` (f-string) across all future Python versions to remain consistent. Not to mention we're not calling `sorted()` on the actual error so can we actually expect it to always be sorted?

We should not care as much about parsing this out precisely imo. We just need to check that the value is in the message or that the right error is raised.

I think for simplicity here maybe we just remove the exact check altogether in this test param and only check `'Expected non-string sequence for "tuple[]" component type: got '`?

Thoughts?